css/ecommerce.css

```diff
@@ -0,0 +1,576 @@
+}
+.header__searchbox::placeholder{
+    text-align: left;
+    font-size: 11px;
+}
+
+
+/* HEADER ENDS HERE */
+
+/* SIGNUP starts here*/
+
+.signup__form{
+   
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    gap: 1rem;
+    padding-top: 5rem;
+    
+}
+.signup__input-field{
+   padding: 1rem;
+}
+
+#signup__name{
+    padding: 10px;
+    border-radius: 5px;
+    height: 35px;
+    background: #F5E8C7;    
+    box-shadow: 1px 1px #0000009d;    
+    outline: none;
+    color: black;
+    border: 1px solid #0000009d   
+}
+.signup__name{    
+    border-radius: 5px;
+    height: 35px;
+    background: #F5E8C7;    
+    box-shadow: 1px 1px #0000009d;    
+    outline: none;
+    color: black;
+    border: 1px solid #0000009d   
+}
+
+.signup__forgoturpw{
+    
+    color: var(--text-color);
+}
+.signup__forgoturpw:hover{
+    
+    text-decoration: underline;
+}
+.signup__btn{
+    width: 15%;
+    height: 40px;
+    padding: 1px;
+    border: none;
+    outline: none;
+    cursor: pointer;   
+    color: #F5E8C7;
+    text-transform: capitalize;
+    font-size: 15px;
+    border-radius: 5px;
+    background: var(--text-color) 
+}
+
+/* SIGNUP ends here*/
+
+/* MAIN STARTS HERE */
+.main{
+    background-color:#F5E8C7 ;
+    height: 150vh;
+}
+
+.cards{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    flex-wrap: wrap;    
+}
+
+.card__discount-tag{
+    position: absolute;
+    padding: 5px;
+    border-radius: 5px;
+    color: #F5E8C7;
+    right: 10px;
+    top: 10px;
+    text-transform: capitalize;
+    font-size: 30px;
+    background: var(--text-color)
+}
+.card__info{
+    display: flex;   
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;   
+}
+
+.card__img{
+    position: relative;
+    width: 100%;
+    height: 350px;
+    overflow: hidden;
+    filter:drop-shadow(3px 5px 2px rgba(62, 61, 61, 0.4))         
+}
+img{
+    padding-top: 2rem;
+    width: 400px;
+    height: 500px;
+    object-fit: cover;
+    
+}
+
+.card__btn{
+    position: absolute;
+    bottom: 10px;
+    left: 10%;
+    transform: translateX(-50%);
+    bottom: 10px;
+    padding: 10px;
+    width: 20%;
+    text-transform: capitalize;
+    border: none;  
+    background: var(--text-color);
+    border-radius: 3px;
+    cursor: pointer;
+    opacity: 1;
+}
+.card__btn:hover{
+    background: #9E7777;
+}
+
+.card__name{
+    font-size: 25px;
+    padding-top: 8px;
+    letter-spacing: 0.5mm;
+    color:var(--text-color)
+    
+}
+.card__description{
+    color: var(--text-color);
+    font-style: italic;
+    font-size: 12px;
+}
+.card__price{    
+    font-weight: 900;
+    font-size: 20px;
+    color:var(--text-color)
+}
+
+.card__actual-price{   
+    margin-left: 20px;
+    opacity: 1;
+    text-decoration: line-through;
+    color:var(--text-color)
+}
+
+/* MAIN ENDS HERE */
+
+/* FOOTER STARTS HERE */
+
+.footer{
```

Comment:
you can make footer stick to bottom  by using:
body{
    min-height: 100vh;
    display: flex;
    flex-direction: column;
}
footer{
    margin-top: auto;
}
Also on mobile it has a scroll, which is not the best